kubernetes/skaffold/src/main/kotlin/com/google/kubernetes/tools/skaffold/run/SkaffoldRunConfigurations.kt

```diff
@@ -123,10 +123,16 @@ abstract class AbstractSkaffoldRunConfiguration(
         if (!SkaffoldExecutorService.instance.isSkaffoldAvailable()) {
             throw RuntimeConfigurationWarning(message("skaffold.run.config.not.on.system.error"),
                     Runnable {
-                        ShowSettingsUtil.getInstance()
-                                .showSettingsDialog(
-                                        project,
-                                        KubernetesSettingsConfigurable::class.java)
+                        val kubernetesConfigurable =
+                                Configurable.APPLICATION_CONFIGURABLE.extensionList.find {
+                            it.id == "google.kubernetes.settings"
```

Comment:
Could we make it a constant? And it's not accessible directly from the class, right?

Reply:
will do.

> And it's not accessible directly from the class, right?

No, its not. This was part of the work to cut out the circular dependency. The Configurable lives in the root module (so the submodules don't have access to it since the dependencies flow in the other direction).

Reply:
done